### DISCUSSION TITLE
No module named 'agents.navigation'; 'agents' is not a package

### DISCUSSION THREAD

**Discussion by Seazoned** [carla-simulator/carla#6864] [Q&A]
![image](https://github.com/carla-simulator/carla/assets/146093507/02b13960-c829-4843-af17-f8490a72a0ff)
When I write this code and try to compile it, I get this error. How can I solve it?

**Comment by Jinviz** [comment]
I need to know the location of the file with the code to solve it.

**Comment by Daraan** [comment]
You need to make sure that _agents _ is in your (PYTHON)PATH variable or your working dictionary.

Setting your working dictionary to `<CARLA_ROOT>/PythonAPI/carla` would fix it as `agents` is a sub dictionary. Similarly adding the carla folder to your PYTHONPATH (quick and dirty way via `sys.path.append("<CARLA_ROOT>/PythonAPI/carla"`) at the top of your file.
